geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/data/AccumuloDataStoreFactory.scala

```diff
@@ -89,8 +92,24 @@ object AccumuloDataStoreFactory {
     val instance = instanceIdParam.lookup[String](params)
     val user = userParam.lookup[String](params)
     val password = passwordParam.lookup[String](params)
+    val keytabPath = keytabPathParam.lookup[String](params)
+
+    // Build authentication token according to how we are authenticating
+    val authToken : AuthenticationToken = if (password != null && keytabPath == null) {
+      new PasswordToken(password.getBytes("UTF-8"))
+    } else if (password == null && keytabPath != null) {
+      if(!useMock) {
```

Comment:
picky - add a space after if but don't let it delay merge we can update it in the upcoming decade :)